README.md

```diff
@@ -1,5 +1,13 @@
 # Balatro-GBA
 
+---
+
+<a href="https://github.com/cellos51/balatro-gba/releases/">
+  <img src="https://img.shields.io/badge/Download_the_ROM_from_github-8A2BE2?&logo=github" alt="Download ROM" width="350">
+</a>
+
+---
```

Comment:
@MeirGavish This is one change I wanted to run by you before I merged. I can drop the commit if you want, but I just put a big ol' download button in the README.